spec/spec_helper.rb

```diff
@@ -16,6 +16,12 @@
 ENV['RACK_ENV'] = 'test'
 $LOAD_PATH.unshift File.join(__dir__, 'lib')
 
+# Disables internationalized strings, which shouldn't be needed for tests.
+# This gets around an issue where Puppet::Environment and GettextSetup in
+# r10k are fighting over the same text domain within the FastGettext domain.
+# https://github.com/voxpupuli/ra10ke/issues/39
+Puppet[:disable_i18n] = true
```

Comment:
This makes me nervous, mostly because I don't know very much about it. Will users run into the gettext error in the wild? I can't reproduce it locally, but it's hard to be 100% sure. Is disabling gettext in Pal an option?

Reply:
Considering the error is caused by a conflict between r10k and `Puppet::Environment`, which is instantiated with `Puppet::Pal.in_tmp_environment`. I doubt this particular error will pop up for users since r10k is never required at the same time a `Puppet::Environment` is instantiated.